whiskey.yml

```diff
@@ -26,21 +26,17 @@ Build:
     Script: Tests\*.Tests.ps1
     Verbose: false
 
+- PowerShell:

```

Comment:
See https://github.com/webmd-health-services/Template-PSModule-OSS-Apache2.0-AppVeyor/blob/main/whiskey.yml for the default build script.